Archit/Source/Films/FilmsViewController.swift

```diff
@@ -108,6 +104,10 @@ class FilmsViewController: BaseViewController<FilmsController> {
         }
     }
 
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 128.0
```

Comment:
It's necessary? If we only have one cell type we can define its height in the storyboard prototype or in the xib with autolayout, right?

Reply:
Yes, it is necessary due to a bug. We've tried to do it on storyboard and it's failing.
https://stackoverflow.com/questions/8615862/custom-cell-row-height-setting-in-storyboard-is-not-responding